src/Pact/Native.hs

```diff
@@ -18,8 +18,16 @@
 module Pact.Native
     (natives
     ,nativeDefs
-    ,moduleToMap)
-    where
+    ,moduleToMap
```

Comment:
Unfortunately I now have to export all the definitions we use to construct terms in `Analyze.Translate`.